### PR TITLE
Use only conda-forge for building the binary Khiops Conda packages

### LINKED ISSUE #388: Build the Conda Packages Using Only the `conda-forge` Channel
#### Description
Currently, the `khiops-core` Conda package (but also KNI) are built using
- the Conda default channels for Windows
- the Conda default channels, plus `conda-forge` for UNIXes (all architectures).

#### Questions/Ideas
- Build the Conda packages using solely the `conda-forge` channels, for all target OSes and architectures

### Context
- Khiops version: >= 10
- Log file (use `khiops -e log.txt`): not applicable
- Scenario file (use `khiops -o scenario._kh`): not applicable
- OS description (use `khiops -s`): Conda on all supported OSes and architectures

### PR DESCRIPTION
**Pull request by popescu-v**
Related issue https://github.com/KhiopsML/khiops-python/issues/189

Closes #388

### CODE REVIEW

File: packaging/conda/bld.bat [hunk 1]
```diff
@@ -2,7 +2,7 @@ REM Echo all output
 @echo on
 
 REM Build the Khiops binaries
-cmake --preset windows-msvc-release -DBUILD_JARS=OFF -DTESTING=OFF
+cmake --preset windows-msvc-release -DBUILD_JARS=OFF -DTESTING=OFF -A "" -T ""
```
Comment (folmos-at-orange) [inline packaging/conda/bld.bat:9]: I'd leave a comment on why we need these options.

File: src/Learning/KWUtils/KWKhiopsVersion.h [hunk 2]
```diff
@@ -10,7 +10,7 @@
 // dans le TaskManager de Windows (par exemple)
 
 // Version de Khiops
-#define KHIOPS_VERSION KHIOPS_STR(10.2.2)
+#define KHIOPS_VERSION KHIOPS_STR(10.3.0-a.4)
```
Comment (folmos-at-orange) [inline src/Learning/KWUtils/KWKhiopsVersion.h]: Why are we incrementing the minor? I think these changes are patches.

**Review by folmos-at-orange — APPROVED** [review]

File: packaging/conda/bld.bat (hunk 1 above)
Comment (popescu-v) [reply packaging/conda/bld.bat:9]: OK, I'd copy the commit message as a comment here.

File: src/Learning/KWUtils/KWKhiopsVersion.h (hunk 2 above)
Comment (popescu-v) [reply src/Learning/KWUtils/KWKhiopsVersion.h]: Yes, this is just a temporary tag for tests only. I think that the "final" tag would depend on where this change would be put on the release roadmap.

**Review by bruno-at-orange — APPROVED** [review]
LGTM

File: src/Learning/KWUtils/KWKhiopsVersion.h (hunk 2 above)
Comment (folmos-at-orange) [reply src/Learning/KWUtils/KWKhiopsVersion.h]: Ok

Comment (popescu-v) [reply src/Learning/KWUtils/KWKhiopsVersion.h]: This should land into 10.2.3 IMHO (if LearningTest reports are OK for all supported platforms).

Comment (popescu-v) [reply src/Learning/KWUtils/KWKhiopsVersion.h]: Tagged with 10.2.3-b.1

File: packaging/conda/bld.bat (hunk 1 above)
Comment (popescu-v) [reply packaging/conda/bld.bat:9]: Done.

**Review by bruno-at-orange — APPROVED** [review]
LGTM

**Pull request merged**